server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -544,7 +544,13 @@ def delete_model_endpoints_resources(
                         project=project_name
                     ),
                 )
-
+            except mlrun.errors.MLRunInvalidArgumentError as e:
+                logger.warning(
+                    "Failed to delete TSDB resources, you may need to delete them manually",
```

Comment:
A bit nitpicky, but all the other errors say `Can't delete` and this one says `Failed to delete`.